backend/geonature/core/gn_synthese/routes.py

```diff
@@ -932,7 +925,7 @@ def get_taxa_count():
 
     if "id_dataset" in params:
         query = query.filter(Synthese.id_dataset == params["id_dataset"])
-    return query.one()
+    return query.one()[0]
```

Comment:
Comme ça (non testé) ?
```suggestion
    return query.scalar()
```

Reply:
Je croyais l'avoir remodifier, mais oui c''est mieux avec scalar